app/models/fs/api_connection.rb

```diff
@@ -68,4 +68,13 @@ def boxify
 
     count
   end
+
+  private
+
+  def generate_short_name_from_name(name)
+    generated_short_name = "FS" + name.split.map(&:first).join.upcase
+    generated_short_name += tenant.boxes.where(short_name: generated_short_name).count.to_s if tenant.boxes.where(short_name: generated_short_name).any?
```

Comment:
nepadne to pri treťom boxe? Máme boxy FSAA a FSAA1. Príde zase AA. generated_short_name = FSAA; počet boxov, kde je FSAA == 1; Vyorbím nový zase FSAA1, nie 2, máme konflikt.

Reply:
Upravene, pozri @celuchmarek

Reply:
Jo, už to vyzerá, že to bude fungovať, keď sa budú generovať iba sekvenčne.

Vidím ale v GO adminovi, že user si vie zmeniť skratku boxu - s týmto nechceme rátať? V tom prípade viem mať vytvorený FSAA a FSAA2, ak sa bude chcieť vytvoriť ďalší FSAA, výjde na neho 2, čo už bude obsadená.

Je to ale pomerne edgecase, tak len dávam do pozornosti a na zváženie @luciajanikova . Inak sa to dá riešiť asi už iba while cyklom.

Reply:
@luciajanikova suhlasim s marekom, dajme tam normalne check, ze ci taky nazov uz existuje vo while a vyriesi nam to vsetky okrajove srandy s premenovavanim.

Reply:
Hej, upravila som